tests/sentry/api/test_organization_events.py

```diff
@@ -0,0 +1,160 @@
+from unittest import mock
+
+from django.urls import reverse
+
+from sentry.search.events import constants
+from sentry.testutils.cases import APITestCase
+from sentry.testutils.helpers.datetime import before_now, iso_format
+from sentry.testutils.silo import region_silo_test
+from sentry.utils.snuba import QueryExecutionError, QueryIllegalTypeOfArgument, RateLimitExceeded
+
+MAX_QUERYABLE_TRANSACTION_THRESHOLDS = 1
+
+
+class OrganizationEventsEndpointTestBase(APITestCase):
```

Comment:
Is there a need to have this as a separate base class instead of just merging it with `OrganizationEventsEndpointTest`?